rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -144,7 +144,9 @@ def add(self, batch: SampleBatchType, **kwargs) -> None:
         warn_replay_capacity(item=batch, num_items=self.capacity / batch.count)
 
         if self._storage_unit == StorageUnit.TIMESTEPS:
-            self._add_single_batch(batch, **kwargs)
+            timeslices = batch.timeslices(1)
+            for t in timeslices:
```

Comment:
Just a question: Could this be anyhow slower than what we had before, b/c we have to slice? I wouldn't think so (b/c we also split into single timesteps before, correct?)

Reply:
We don't split before - actually what arrives at add() are just rollout batches.
So I think this could be a little slower. But for example in the context of long rollouts and PER, I think we would miss out on policy improvement per sample if we moved everything to storage_units="fragments" by default.

Reply:
Except for the case of a ReplayBuffer being an underlying replay buffer of a MultiAgentReplayBuffer, in which case experiences are timesliced before already.

Reply:
So imho this is safe to merge.